configure.ac

```diff
@@ -1,5 +1,5 @@
 # Motr version
-m4_define([M0_VERSION_MAJOR],[1])
+m4_define([M0_VERSION_MAJOR],[2])
```

Comment:
Can we also accept version number as input from RE and use the same. If those values are not passed, we can use these values as default.

Reply:
OK. This will be done in another PR.

Reply:
https://github.com/Seagate/cortx-motr/pull/422